storage/pkg/flagset/storageoc.go

```diff
@@ -1,116 +0,0 @@
-package flagset
-
-import (
-	"github.com/micro/cli/v2"
-	"github.com/owncloud/ocis/storage/pkg/config"
-)
-
-// StorageOCWithConfig applies cfg to the root flagset
-func StorageOCWithConfig(cfg *config.Config) []cli.Flag {
-	flags := []cli.Flag{
-
-		// debug ports are the odd ports
-		&cli.StringFlag{
-			Name:        "debug-addr",
-			Value:       "0.0.0.0:9163",
-			Usage:       "Address to bind debug server",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DEBUG_ADDR"},
-			Destination: &cfg.Reva.StorageOC.DebugAddr,
-		},
-
-		// Services
-
-		// Storage oc
-
-		&cli.StringFlag{
-			Name:        "network",
-			Value:       "tcp",
-			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_NETWORK"},
-			Destination: &cfg.Reva.StorageOC.Network,
-		},
-		&cli.StringFlag{
-			Name:        "protocol",
-			Value:       "grpc",
-			Usage:       "protocol for storage service, can be 'http' or 'grpc'",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_PROTOCOL"},
-			Destination: &cfg.Reva.StorageOC.Protocol,
-		},
-		&cli.StringFlag{
-			Name:        "addr",
-			Value:       "0.0.0.0:9162",
-			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_ADDR"},
-			Destination: &cfg.Reva.StorageOC.Addr,
-		},
-		&cli.StringFlag{
-			Name:        "url",
-			Value:       "localhost:9162",
-			Usage:       "URL to use for the storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_URL"},
-			Destination: &cfg.Reva.StorageOC.URL,
-		},
-		&cli.StringSliceFlag{
-			Name:    "service",
-			Value:   cli.NewStringSlice("storageprovider"),
-			Usage:   "--service storageprovider [--service otherservice]",
-			EnvVars: []string{"STORAGE_STORAGE_OC_SERVICES"},
-		},
-
-		&cli.StringFlag{
-			Name:        "driver",
-			Value:       "owncloud",
-			Usage:       "storage driver for oc mount: eg. local, eos, owncloud, ocis or s3",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DRIVER"},
```

Comment:
used in drone https://github.com/owncloud/ocis/blob/d422a91a01b98893d0a10a7555092342755ee33b/.drone.star#L548
not needed anymore?

Reply:
Yes, the storages got streamlined so there is no OC storage anymore.